examples/flows/standard/analyze_documents/README.md

```diff
@@ -0,0 +1,63 @@
+# Analyze Documents
```

Comment:
Could you move your samples to below folder?
examples/flows/integrations/azure-ai-language